### ISSUE TITLE
File to import not found or unreadable: bootstrap

### ISSUE THREAD

**Issue by manuel-reil** [mathieul/ember-cli-compass-bootstrap#3]
Hi,

I strictly followed the readme, but while executing "ember server" I receive the error: "error app/styles/app.scss (Line 2: File to import not found or unreadable: bootstrap."

compass seems to load, though.

Brocfile.js:

```
/* global require, module */

var EmberApp = require('ember-cli/lib/broccoli/ember-app');
var pickFiles  = require('broccoli-static-compiler');
var mergeTrees = require('broccoli-merge-trees');

var app = new EmberApp();

// Use `app.import` to add additional libraries to the generated
// output files.
//
// If you need to use different assets in different
// environments, specify an object as the first parameter. That
// object's keys should be the environment name and the values
// should be the asset to use in that environment.
//
// If the library that you are including contains AMD or ES6
// modules that you would like to import into your application
// please specify an object with the list of modules as keys
// along with the exports of each module as its value.

var bootstrapDir = 'bower_components/bootstrap-sass-official/assets';

// select bootstrap JavaScript components to include
var bootstrapComponents = ['dropdown', 'alert'];

for (var index in bootstrapComponents) {
  app.import(bootstrapDir + '/javascripts/bootstrap/' + bootstrapComponents[index] + '.js');
}

var extraAssets = pickFiles(bootstrapDir + '/fonts/bootstrap', {
  srcDir: '/',
  destDir: '/assets/bootstrap'
});

module.exports = mergeTrees([app.toTree(), extraAssets]);
```

app.scss:

```
@import "bootstrap";

// Compass documentation: http://compass-style.org
@import "compass";

body {
  margin-top: 50px;
}

.navbar-form .form-group {
  display: inline-block;
  margin-bottom: 0;
  vertical-align: middle;
}
```

Thank you for your help,
Manuel

**Comment by mathieul** [comment]
Hi Manuel. I've not been able to reproduce your problem. I've created a brand new application using ember-cli v0.1.5 and followed each step and all work fine.

I've just published a _0.0.6_ version which doesn't use a hard-coded path for the application bower sub-folder, that might fix your problem if the path is the cause. I also documented in the last section of the README file an alternative to using this package which could also help you.

I hope that helps.
